examples/acceptFromIDs.js

```diff
@@ -0,0 +1,63 @@
+// Required Modules
+const SteamUser = require('steam-user'),
+const SteamTotp = require('steam-totp'),
+const TradeOfferManager = require('steam-tradeoffer-manager'),
+const SteamCommunity = require('steamcommunity');
+ 
+// Initiate Modules
+let client = new SteamUser();
+let community = new SteamCommunity(),
+let manager = new TradeOfferManager({
+    "steam": client,
+    "domain": "nothinghere",
+    "language": "en"
+});
+ 
+let config = {
```

Comment:
In all other examples `logOnOptions` is used. How about sticking with that:

```JS
// Steam logon options
var logOnOptions = {
	"accountName": "username",
	"password": "password",
	"twoFactorCode": SteamTotp.getAuthCode("sharedSecret")
};
```

and separated variable with accepted `IDs`